docs/2.0/admin-guide.md

```diff
@@ -909,6 +909,23 @@ $ ssh-keygen -L -f ~/.tsh/keys/localhost/jsmith.cert
                 permit-pty
 ```
 
+### HTTP CONNECT Proxy
+
+Some networks funnel all connections through a proxy server where they can be
+audited and access control rules applied. For these scenarios Teleport supports
+HTTP CONNECT tunneling.
+
+To use HTTP CONNECT tunneling, simply set either the `https_proxy` or
```

Comment:
are those standard names for env variables? generally linux env vars are using `HTTPS_PROXY` upper case notations, what are others are using?